prefs.js

```diff
@@ -31,7 +39,44 @@ function buildPrefsWidget() {
     });
     box.add(label);
     box.add(button);
+    box.add(new Gtk.Separator({}));
+    box.add(buildNichePrefsWidget());
+
     box.show_all();
 
     return box;
 }
+
+function buildNichePrefsWidget() {
+    const box = new Gtk.Box({
+        orientation: Gtk.Orientation.VERTICAL,
+        spacing: 9,
+        halign: Gtk.Align.FILL,
+        valign: Gtk.Align.CENTER,
+    });
+    box.add(new Gtk.Label({
+        label: "<b>Niche Settings</b>",
+        justify: Gtk.Justification.CENTER,
+        use_markup: true,
+    }));
+
+    const superKeyBox = new Gtk.Box({
+        orientation: Gtk.Orientation.HORIZONTAL,
+        spacing: 9,
+        halign: Gtk.Align.FILL,
+        valign: Gtk.Align.CENTER,
+    });
+    superKeyBox.add(new Gtk.Label({
+        label: "Disable super key action (reverts to gnome default)",
```

Comment:
If this label was going to be present, GNOME would need to be capitalized. We also typically capitalize `Super key` in copy.

Reply:
Fixed in https://github.com/pop-os/cosmic/pull/183/commits/bc95be0445eef490166697af73e3e3ce6ac094db.